crowbar_framework/lib/crowbar/logger/formatter.rb

```diff
@@ -18,20 +18,55 @@ module Crowbar
   module Logger
     class Formatter < ::Logger::Formatter
       include ActiveSupport::TaggedLogging::Formatter
+      FORMAT = "%s, [%s#%d] %5s -- %s\n"
+      THREAD_FORMAT = "%s, [%s#%d:%s] %5s -- %s\n"
 
-      def call(severity, timestamp, progname, msg)
-        final_message = if msg.is_a? String
+      def call(severity, time, progname, msg)
+        if (ENV["CROWBAR_THREADS"].to_i || 10) > 1
```

Comment:
maybe a more explicit approach? 10 is kind of irrelevant here:

```ruby
threads = ENV["CROWBAR_THREADS"]
if threads.to_s.empty? || threads.to_i > 1?
  ...
else
  ...
end
```

And you are right!

```
irb(main):001:0> 0 || 10
=> 0
```

but this looks like works:

```
irb(main):005:0> 0 | 10
=> 10
```